engine/src/main/java/org/camunda/bpm/engine/task/Task.java

```diff
@@ -189,4 +189,9 @@ public interface Task {
    */
   void setTenantId(String tenantId);
 
+  /** Returns if an attachment exists for the task */
+  boolean getAttachments();
+  /** Signifies if a comment exists for the task */
+  boolean getComments();
```

Comment:
🔧 Following the Java convention, let's use `has` for the boolean getters.
```suggestion
  /** Returns if an attachment exists for the task */
  boolean hasAttachment();
  /** Signifies if a comment exists for the task */
  boolean hasComment();
```